try {
	[INSERT]
} catch (error:Error) {
	
}